Procfile

```diff
@@ -1,2 +1,3 @@
-vue: vue-cli-service build --mode development --watch
+#vue: vue-cli-service build --mode development --watch
+vue: vue-cli-service serve
```

Comment:
Did `serve` instead of `build` end up working better for you? Have you had the chance to try #805?